mayastor-test/e2e/common/io_connect_node.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Script to disconnect a node from another node using iptables
+# $1 is the node-name to isolate/restore
+# $2 is the other node-name
+# $3 is "I" to disconnect, "D" to reconnect
+# assumes the nodes use a known fixed set of IP addresses and node names
+
+# edit the line below, if necessary
+KUBESPRAY_REPO="$HOME/work/kubespray"
+
+if [ $# -ne 3 ];
+    then echo "specify node-name, other node-name  and action (D or I)"
+    exit 1
+fi
+
+cd ${KUBESPRAY_REPO}
+
+nodename=$1
+other_nodename=$2
+other_node_suffix=${other_nodename: -1}
+
+other_ip=172.18.8.10${other_node_suffix}
+
+# apply the rule to block/unblock it
+vagrant ssh ${nodename} -c "sudo sh -c 'sudo iptables -$3 INPUT -s ${other_ip} -j REJECT'"
+vagrant ssh ${nodename} -c "sudo sh -c 'sudo iptables -$3 OUTPUT -s ${other_ip} -j REJECT'"
```

Comment:
`REJECT` and `DROP` produce different behavior (REJECT sends ICMP, DROP is just silent) and it might be worth testing both scenarios.

Is sudo really needed twice? I imagine either of them would be enough.

Reply:
Yes, DROP will be a separate case, to be added later. I'll remove the extra sudo.